### DISCUSSION TITLE
posts/atomic-kotlin/

### DISCUSSION THREAD

**Discussion by giscus** [ddiyooong/ddiyooong.github.io#1] [General]
# posts/atomic-kotlin-1/

아토믹 코틀린을 읽으면서 새로 알게 된 내용, 애매한 부분을 정리함

http://127.0.0.1:4000/posts/atomic-kotlin/

**Comment by ddiyooong** [comment]
test

**Reply by kth990303** [reply]
이 테스트는 뭐죠? ㅋㅋㅋㅋ

**Reply by ddiyooong** [reply]
zzㅋㅋㅋㅋ 깃헙 블로그 처음 만들어봐서 giscus 테스트 중입니다 잘되네요 ^^7

**Comment by kth990303** [comment]
아토믹 코틀린 책 읽어보고 싶었는데 요약본을 올려주셨군요
앞으로도 자주 올라오나요? ㅎㅎ
잘읽었습니다

**Reply by ddiyooong** [reply]
역시 케이티에잍치구구공삼공삼... 블로그 생기자마자 오시다니 인스타 대신 깃허브 보는거 딱들킴 ㅋ 앞으로 여기에 2~8부 추가할 거니까 많이 봐주세요 ^^7
